src/java.base/share/classes/jdk/internal/module/ModuleBootstrap.java

```diff
@@ -463,18 +464,21 @@ private static ModuleLayer boot2() {
 
         // Step 8: CDS dump phase
 
-        if (CDS.isDumpingStaticArchive() && !haveUpgradeModulePath && addModules.isEmpty()) {
+        if (CDS.isDumpingStaticArchive()
+                && !haveUpgradeModulePath
+                && addModules.isEmpty()
+                && allJrtOrModularJar(cf)) {
             assert !isPatched;
 
             // Archive module graph and maybe boot layer
             boolean hasSplitPackages = containsSplitPackages(cf);
             boolean hasIncubatorModules = containsIncubatorModule(cf);
             ArchivedModuleGraph.archive(hasSplitPackages,
-                                        hasIncubatorModules,
-                                        systemModuleFinder,
-                                        cf,
-                                        clf,
-                                        mainModule);
+                    hasIncubatorModules,
+                    systemModuleFinder,
+                    cf,
+                    clf,
+                    mainModule);
```

Comment:
This was correctly aligned before, now it isn't.

Reply:
Fixed.

Reply:
That may have been my fault, I gave Calvin changes to check the configuration and the IDE re-formatted it.